src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Kindly rename your action types here in keeping with the ducks pattern `(app/reducer/ACTION_TYPE)`. For example: `const ADD_BOOK = "bookstore/books/ADD_BOOK"`